symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -96,6 +99,13 @@ protected SymphonyBdk(BdkConfig config, ApiClientFactory apiClientFactory)
     // setup activities
     this.activityRegistry =
```

Comment:
would it not be better to have proper ifs here? to follow the flow
if we have a DF loop then 
     we create a registry
     if we have a message service
        then we add the help command to it